duckduckgo/Android
# Amend translations

Task/Issue URL: https://app.asana.com/0/1125189844152671/1188623309021366
Tech Design URL: 
CC: 

**Description**:
This PR adds the new translations from Smartling. There are quite a few changes which include:
- In all the translations the placeholders have been replaced with the type `%1$s`
- Some strings have been amended to remove quotes that were put there by mistake or to remove unnecessary spaces

**Steps to test this PR**:
Check different languages to make sure everything looks good.

Double check the following:
1. Spacing and break lines in settings/about DuckDuckGo look good.
1. Go to `cnn.com` make sure the break lines are good and the dialog shows the emoji correctly.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
